.github/workflows/php-static-analysis.yml

```diff
@@ -33,35 +33,20 @@ jobs:
         with:
             PHP_MATRIX: '["8.0", "8.1", "8.2"]'
 
-    coding-standards-analysis-php-80:
+    coding-standards-analysis-php:
         if: ${{ (github.event_name != 'workflow_dispatch') || ((github.event.inputs.jobs == 'Run all') || (github.event.inputs.jobs == 'Run PHPCS only')) }}
         uses: inpsyde/reusable-workflows/.github/workflows/coding-standards-php.yml@main
+        strategy:
```

Comment:
PHPCS does not need to run in a matrix. It will return the same result for all versions.

Reply:
changed